client-app/ui-kit/composables/useQuantityValidationSchema.test.ts

```diff
@@ -0,0 +1,121 @@
+import { describe, expect, it } from "vitest";
```

Comment:
To simplify comparison with diagram, I will suggest to add some missed tests & reorder these tests in the following order:
1. positive (exists)
2. integer (exists)
3. available < min (exists)
4. limit (exists)
5. min < available (exists)
6. min < max (exists)
7. min & available & max
    1. min < available < max **(not exists)**
    2. min < max < available (exists)
8. min (exists)
9. available (exists)
10. max (exists)
11. available & max
    1. available < max (exists)
    2. max < available (exists)


![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/6369252/a0de332e-faf7-4236-a1bd-bc4fd48385bb)